code/__DEFINES/components.dm

```diff
@@ -23,6 +28,7 @@
 
 // /atom signals
 #define COMSIG_PARENT_ATTACKBY "atom_attackby"			        //from base of atom/attackby(): (/obj/item, /mob/living, params)
+	#define COMPONENT_NO_AFTERATTACK 2								//Return this in response if you don't want afterattack to be called

```

Comment:
indentation?

Reply:
intentional

Reply:
why

Reply:
It doesn't really matter too much but it feels like the best way given the circumstances to indicate sub-grouping. The other option is to add more newlines and/or comments. This is already a heavily commented file, and newlines would effectively double the length in order to maintain consistency.

It looks a little dumb but it's clear what's happening here and doesn't waste space.